Applications/JupyterHub.qmd

```diff
@@ -3,72 +3,47 @@ title: "JupyterLab"
 
-You can access the JupyterLab service by clicking on this link.: [https://jupyterhub.dataspace.copernicus.eu/](https://jupyterhub.dataspace.copernicus.eu/){target="_blank"}
+## Notebooks
 
-![](_images/JupyterLab_welcome.png)
+Within JupyterLab, Python notebooks can be used as an interactive programming environment for those who want to prototype their EO data processing quickly. Additionally, example notebooks are available to help users get started with their analysis. 
 
-Upon clicking the "ACCESS JUPYTERLAB" button, you'll be directed to the login window. Please use your Copernicus Data Space Ecosystem credentials.
+Access to this service is available by clicking on this link: [https://jupyterhub.dataspace.copernicus.eu/](https://jupyterhub.dataspace.copernicus.eu/){target="_blank"}
 
-## Notebooks
+![](_images/JupyterLab_welcome.png)
 
-Notebooks in JupyterLab are an interactive Python programming environment for both non-programmers and developers who want to quickly prototype their EO data processing. 
-They provide a user-friendly interface, integrating well with services like Sentinel Hub and openEO. 
-Each notebook has direct access to the EO Data repository, and example notebooks are available to facilitate data interaction. 
-To handle tool dependencies and library versions, multiple notebook kernels are provided and regularly updated with the provided samples. 
-This eliminates the need for users to handle dependency installations, enabling immediate prototyping without technical obstacles.
+Upon clicking the "ACCESS JUPYTERLAB" button, users will be directed to the login window. Please use Copernicus Data Space Ecosystem credentials to log in.
 
 
-## Notebook flavors
+## Server Options
 
-When you log in to JupyterLab you are presented with a choice of 3 flavors of Jupyter instances: Small, Medium and Large.
-The size of the instance is determined by number of resources available to the notebook kernels run by the user - CPU cores and memory.
-All flavors are suitable for performing typical tasks and will be capable of running all samples provided in /samples folder.
-To ensure fair use of resources by the CDSE users, it is recommended to start with the Small flavor and switch to a bigger only when you experience issues with kernel crashing due to the lack of available memory.
+Once logged in to JupyterLab, users are offered a choice of 3 flavors of Jupyter instances: Small, Medium and Large.
+The size of the instance is based on how many resources, like CPU cores and memory, are allocated to the notebook kernels.
+All flavours are suitable for performing simple tasks and can run all samples provided in the samples folder. For fair resource usage, it is suggested to start with the "Small" flavour and switch to a larger one if kernel crashes is encounted caused by insufficient memory.
```

Comment:
is there also an implication for credit cost when running operations? Do the large ones consume more?

Reply:
No, they don't. These are still free services. But do you suggest mentioning that, too?

Reply:
no, I would not add additional information. I was simply wondering if there were any real implications